ldk/javascript/src/whisper/react/whisper-render-instance.ts

```diff
@@ -0,0 +1,58 @@
+import { NewWhisper, UpdateWhisper, Whisper } from '../types';
+import { create } from '../index';
+
+export interface WhisperRenderingInterface {
+  createOrUpdateWhisper(whisperData: NewWhisper | UpdateWhisper): void;
+  closeWhisper(): void;
+  setOnClose(func: () => Promise<void>): void;
+}
+
+enum RenderInstanceStatus {
+  NotCreated,
+  Created,
+  Closed,
+}
+
+/**
+ * @internal
+ */
+export class WhisperRenderInstance implements WhisperRenderingInterface {
+  private status: RenderInstanceStatus = RenderInstanceStatus.NotCreated;
+
+  private whisper: Whisper | undefined;
+
+  private onCloseHandler: undefined | (() => Promise<void>);
+
+  async createOrUpdateWhisper(whisperData: NewWhisper | UpdateWhisper): Promise<void> {
+    if (this.status === RenderInstanceStatus.Closed) {
+      return;
```

Comment:
Should this do something harsher than just a return? Below, if the status is in an invalid state then we throw an error. Would this be considered an invalid state?

Reply:
This was an interesting area and I'm not 100% sure what to do yet. 

When I'm unmounting components after the Whisper has been closed, I'm still getting a final call to this function to update the container with no components.  I don't see a way to "destroy" a container.